### PR TITLE
Update pto-request-approval-process.md

### PR DESCRIPTION
**Pull request by hhartingrex**
[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/hhartingrex-patch-2/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

**Comment by hhartingrex** [comment]
Suggested adjustments, as per the Project Management Practice Area. Thank you!

**Review by farooqzakhilwal — COMMENTED** [review]
Looks good. Over to @akaroleff for approval.

**Comment by hhartingrex** [comment]
NOTE: This page conflicts with https://handbook.civicactions.com/en/latest/040-employee-handbook-us/benefits-and-holidays/#holidays
Are we able to only have one page?

**Review by akaroleff — COMMENTED** [review]
This seems duplicative with slightly different info from the benefits page about requesting time off. Perhaps ask Marc if he's ok with having this page too and if so... maybe we should frame this from the Project Manager's seat about how to set expectations with their team for requesting time off?

**Review by farooqzakhilwal — COMMENTED** [review]
@hhartingrex this looks great. Should we have an additional option under the unexpected event process where an individual may not be able to access Unanet and other tools required? Perhaps they should inform their AD or direct line manager?

**Review by dmundra — APPROVED** [review]
@hhartingrex approving. You can click the squash & merge button if you prefer.

File: docs/070-project-management/leave-requests-and-stepping-away.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Leave Requests and Stepping Away: Guidelines for Project Managers
+
+Guidelines that Project Managers may use to coach teams on communications for OOO and stepping away:
+
+## Out Of Office (OOO) Process
```
Comment (grugnog) [inline docs/070-project-management/leave-requests-and-stepping-away.md:5]: @hhartingrex @akaroleff @farooqzakhilwal @dmundra this change seems to have removed anything about project managers pre-approving requests for time off, which in our current process happens before the formal request is placed in Unanet.
This seems like a critical step in the process to me, as project managers are the primary people who would be aware of the impacts requested time off would have on the project in terms of contract deliverables, SLAs, release schedules etc. I think we need to make it clearer that project managers are responsible for managing this, and proactively working with their team to alter time off schedules as needed (which might mean denying requests at times) to ensure projects can continue to meet their obligations.

**Review by grugnog — CHANGES_REQUESTED** [review]

File: docs/070-project-management/leave-requests-and-stepping-away.md (hunk 1 above)
Comment (hhartingrex) [reply docs/070-project-management/leave-requests-and-stepping-away.md:5]: Agreed and thank you @grugnog . Change inserted, hopefully correctly.

**Review by grugnog — APPROVED** [review]

File: docs/070-project-management/leave-requests-and-stepping-away.md (hunk 1 above)
Comment (grugnog) [reply docs/070-project-management/leave-requests-and-stepping-away.md:5]: I think probably more detail we could add here, but that covers it for now I think - thanks!

**Pull request merged**